ansible/localhost/virtualhost.conf

```diff
@@ -0,0 +1,24 @@
+<IfModule mod_ssl.c>
+
+<VirtualHost *:{{ http_port }}>
+    ServerAdmin webmaster@{{ domain }}
+    ServerName {{ domain }}
+    ServerAlias www.{{ domain }}
+    WSGIScriptAlias / /vagrant/holocron_api/holocron_api/wsgi.py
+
+    Alias /static/  "/vagrant/holocron_api/staticfiles/"
+
+
+    #   SSL Engine Switch:
+    #   Enable/Disable SSL for this virtual host.
+    SSLEngine on
+    SSLCertificateFile /etc/apache2/ssl/server.crt
+    SSLCertificateKeyFile /etc/apache2/ssl/server.key
+
```

Comment:
As mentioned earlier, we should probably be using the snake oil cert as the default:

I believe they're in:
/etc/ssl/certs/ssl-cert-snakeoil.pem
/etc/ssl/private/ssl-cert-snakeoil.key